src/structlog/_log_levels.py

```diff
@@ -91,16 +92,24 @@ def exception(
 async def aexception(
     self: FilteringBoundLogger, event: str, *args: Any, **kw: Any
 ) -> Any:
+    """
+    .. versionchanged:: 23.3.0
+       Implemented `contextvars.ContextVar` for holding and resetting async calling stack
```

Comment:
This is an implementation detail. To the user the relevant part is "Callsite parameters are now also collected under asyncio." or something like that. In that vein: it really just needs an version changed on CallsiteParameters I would say.

Reply:
Makes sense -- updated the lines